No rule to make target `install-openni2'

Hi,
I was trying to follow the installation instruction for Mac. Everything was fine, I successfully finished this part "Run the test program: ./bin/Protonect"

However, I stuck on the following part:
Test OpenNI2. `make install-openni2` (may need sudo), then run `NiViewer`. Environment variable `LIBFREENECT2_PIPELINE` can be set to `cl`,` cuda`, etc to specify the pipeline.

cindydeMacBook-Pro:build cindy$ sudo make install-openni2
make: *** No rule to make target `install-openni2'.  Stop.

Is there anyone know what is going on?
Thank you so much!!

--- a/cmake_modules/FindOpenNI2.cmake
+++ b/cmake_modules/FindOpenNI2.cmake
@@ -35,6 +35,7 @@ FIND_LIBRARY(OpenNI2_LIBRARY
     "/opt/local/lib"
     "/usr/lib"
     "/usr/local/lib"
+    "/usr/local/lib/ni2"
     ENV OPENNI2_REDIST
     ENV PROGRAMFILES
     ENV ProgramW6432

